### DISCUSSION TITLE
Is it possible to collect/click YouTube banner ads?

### DISCUSSION THREAD

**Discussion by ajamico** [dhowe/AdNauseam#2095] [Q&A]
Hi! I know that it's not yet possible to collect or click on youtube's interstitial ads ([as mentioned here](https://github.com/dhowe/AdNauseam/discussions/2060#discussion-3940624)), but is there a way to click on other forms of ads on youtube? I've noticed that there's three types (shown below)

Interstitial ads, the normal type of video ad that most people think of when they hear about ads on youtube:
![YouTube Interstitial Ad](https://user-images.githubusercontent.com/47133932/167493732-8b020b2e-ba9e-422b-82d2-b669fa872125.png)

Banner ads, which pop up on the bottom of a playing video:
![YouTube Banner Ad](https://user-images.githubusercontent.com/47133932/167493757-a021975e-9fc7-4c46-9529-336a556a83a4.png)

and I'm not sure what to call these, but they show up in lists of videos like in the suggested videos and home page:
![YouTube Listing Ad](https://user-images.githubusercontent.com/47133932/167493739-1cb32f07-39ed-4e7d-a4be-1f9286ed812c.png)

I noticed that the banner ads look almost exactly like the type of ad that AdNauseam usually collects. Would it be possible to block the interstitial ads like normal, but collect and click on banner ads (and maybe the listing ads too)?

**Comment by dhowe** [comment]
@ajamico please include a URL for a page with the type of ad shown above
@mneunomne can you check what this (collect/click non-video ads on youtube) would involve?

**Reply by mneunomne** [reply]
Yes I can try, but I would say it's risky. We would need to enable some scripts and rely on the cosmetic filters to both hide and collect these ads. Since sites like youtube, facebook etc are always changing their html structure in relation to ads, it will be difficult to have any solution will be consistent enough and that won't break as soon as Youtube makes some minor tweaks on their code.

**Comment by dhowe** [comment]
But this is true of many sites... the question is whether we can guarantee blocking of video ad content without the strict-block. If so, then the only 'risk' is that we end up with what we have currently, which is not collecting ads
